guacamole-jetty/src/main/java/dev/crafting/guacamole/Parameters.java

```diff
@@ -34,6 +45,14 @@ private Parameters(Parameters from) {
         password = from.password;
     }
 
+    public static String ConfigFile() {
+        String configFile = System.getenv("GUACAMOLE_PARAMETERS");
+        if (configFile == null || configFile.isEmpty()) {
+            configFile = "/etc/guacamole/parameters.properties";
```

Comment:
Inconsistency: in Dockerfile it's `config.properties`, here it's `parameters.properties`. Let's use latter.